Tests/APITesters/ObjCAPITester/ObjCAPITester/RCPurchasesAPI.m

```diff
@@ -31,6 +31,7 @@ @implementation RCPurchasesAPI
 
 + (void)checkAPI {
     RCPurchases *p = [RCPurchases configureWithAPIKey:@""];
+    [RCPurchases configureWithConfiguration:[[RCConfiguration alloc] init]];
```

Comment:
I wonder if the Obj-C method should also be `[RCPurchases configureWith:]`?

Reply:
I don't think so. Usually the method names are extra descriptive of the first parameter.

Reply:
I have your opposite problem haha, I only think in Swift now 😅